server/controllers/AuthorController.js

```diff
@@ -0,0 +1,13 @@
+
+/* eslint-disable require-jsdoc */
```

Comment:
You can't just disable rules you aren't comfortable with. The rule requires that you provide a valid js doc documentation of this class and it's purpose. Please do so.